app/services/DepCodes.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import models.ExportsDeclaration
+
+import java.nio.charset.StandardCharsets.ISO_8859_1
+import java.nio.file.{Files, Paths}
+import scala.collection.JavaConverters.asScalaIteratorConverter
+
+object DepCodes {
+
+  def isDesignatedExportPlaceCode(declaration: ExportsDeclaration): Boolean =
+    declaration.locations.goodsLocation.exists(location => designatedExportPlaceCodes.contains(location.code))
+
+  private val designatedExportPlaceCodes: Set[String] = {
+    val filename = "Designated_Export_Place_codes_for_Data_Element-5-23_of_the_CDS__v2__1_.csv"
```

Comment:
Can this data not be loaded as a JSON file like procedure codes and countries? We are trying move all the existing CSV data into JSON to support the future reference values service.

Reply:
Agreed to replace this CSV file loading with the standard json loading service pattern after the Navigator object has been converted into a singleton. :+1: